modules/security_hub/variables.tf

```diff
@@ -102,3 +102,9 @@ variable "pagerduty_securityhub_integration_key" {
   description = "The PagerDuty integration key to subscribe to SecurityHub findings"
   sensitive   = true
 }
+
+variable "macie_enabled" {
+  type        = bool
+  default     = true
```

Comment:
I don't think we should have a default here, it should be a required input to prevent a change to the module defaulting everyone to enabled